python/usb_console/radio_session.py

```diff
@@ -153,6 +157,9 @@ def read_state(self, field, timeout=None):
         Read state.
         
         Read the latest value of the state field received via downlink with the name "field".
+
+        I can get the latest MTMSN value from the confirmation email through the check downlink function (self.mtmsn).
+        How do I get MTMSN number of the latest uplink?
```

Comment:
Might be a good idea to rename `check_for_downlink` to `check_email` in this case, since the thread would have to do the job of processing both uplink as well as downlink emails. Try to keep everything as logically named as possible when making this change--shoehorning in features reduces readability.